esm/data.py

```diff
@@ -133,11 +135,11 @@ def get_tok(self, ind):
     def to_dict(self):
         return self.tok_to_idx.copy()
 
-    def get_batch_converter(self):
+    def get_batch_converter(self, truncate: bool):
```

Comment:
if we make this argument an `Optional[int]` with default of `None` named something like `truncation_seq_length` - we can pass in the max length at which to truncate. And no truncation by default.